gravityflow/gravityflow
# Add update user step

Adds the Update User Step which updates user meta and properties.

It also removes the generic_map setting as it's now in the add-on framework.

It also adds PHP7.3 to the Travis CI test matrix.

**Testing Instructions**
- Add an Update User Step and map the properties and meta keys to fields or custom values and check that all properties, roles and user meta are set correctly.
- Check each of the different options for identifying the user work as expected.
- The custom meta value supports merge tags and shortcodes e.g. [gv_math scope={entry_id}]{user:rank}+1[/gv_math]
- Unit tests included - check for completeness.
- Check for regressions as a result of switching to the framework version of the generic_map setting.
